src/clj/rems/api/licenses.clj

```diff
@@ -85,7 +85,7 @@
     (POST "/add_attachment" []
       :summary "Add an attachment file that will be used in a license"
       :roles +admin-write-roles+
-      :multipart-params [file :- upload/TempFileUpload]
+      :multipart-params [file :- schema/FileUpload]
       :middleware [multipart/wrap-multipart-params]
```

Comment:
Hmm, was this even necessary before?

Reply:
as far as i understand, nope, since the default middleware intercepted the multipart request before this part. i would guess it went unnoticed since the net effect was the same